redaxo/src/core/lib/api_function.php

```diff
@@ -200,6 +213,17 @@ public function getResult()
     {
         return $this->result;
     }
+
+    /**
+     * This method should be overwritten in all api func classes and the return value should be changed to `true`.
+     * All links for calling the api should then be generated with `your_api_class::getUrlParams()`.
+     *
+     * @return bool
```

Comment:
Csrf validation is disabled by default for backwards compatiblity reasons. This default will change in a future version. Prepare all your api functions to work with csrf token by using `your-api-class::getUrlParams()`, otherwise they will stop work.